tests/js/spec/components/projects/bookmarkStar.spec.tsx

```diff
@@ -1,22 +1,15 @@
-import {mountWithTheme} from 'sentry-test/enzyme';
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
 
 import BookmarkStar from 'sentry/components/projects/bookmarkStar';
 
 describe('BookmarkStar', function () {
-  let wrapper, projectMock;
+  let projectMock: jest.Mock;
 
   beforeEach(function () {
-    wrapper = mountWithTheme(
-      <BookmarkStar
-        organization={TestStubs.Organization()}
-        project={TestStubs.Project()}
-      />
-    );
-
     projectMock = MockApiClient.addMockResponse({
       url: '/projects/org-slug/project-slug/',
       method: 'PUT',
-      data: TestStubs.Project({isBookmarked: false, platform: 'javascript'}),
+      body: TestStubs.Project({isBookmarked: false, platform: 'javascript'}),
```

Comment:
What was changed that triggers this?

Reply:
I changed the file to typescript and this was wrong